ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -13688,6 +13688,22 @@ ASTNode analyzeCreateTable(
       // outputs is empty, which means this create table happens in the current
       // database.
       rootTasks.add(TaskFactory.get(new DDLWork(getInputs(), getOutputs(), crtTblDesc)));
+      String tblLocation = null;
```

Comment:
What happens with the location if the Transaltion kicks in?

Reply:
There is a `Translator` layer in HMS which can change some properties fo the table (not sure which ones), but I remember that this `Translator` was responsible to changing location for the external tables. Does this change able to handle this `Translator`?